pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -218,10 +218,10 @@ func NewFramework(r Registry, plugins *config.Plugins, args []config.PluginConfi
 				if !ok {
 					return nil, fmt.Errorf("plugin %v does not extend queue sort plugin", qs.Name)
 				}
-				f.queueSortPlugins = append(f.queueSortPlugins, p)
```

Comment:
This is intentional. We should be consistent with other plugins and throw an error so that a user knows the problem when they specify more than on queue sort plugin.